website/docs/topics/long_contexts.md

```diff
@@ -62,11 +62,11 @@ By applying the `MessageHistoryLimiter`, we can see that we were able to limit t
 
 #### Example 2: Limiting the Number of Tokens
 
-To adhere to token limitations, use the `MessageTokenLimiter` transformation. This limits tokens per message and the total token count across all messages:
+To adhere to token limitations, use the `MessageTokenLimiter` transformation. This limits tokens per message and the total token count across all messages. Additionally, a min tokens threshold can be applied:
 
 ```python
 # Limit the token limit per message to 3 tokens
-token_limit_transform = transforms.MessageTokenLimiter(max_tokens_per_message=3)
+token_limit_transform = transforms.MessageTokenLimiter(max_tokens_per_message=3, min_tokens=10)
```

Comment:
Show a specific example of how `min_tokens` works. Users will go to this guide for reference.